Solutions/Corvus.Testing.AzureFunctions.Xunit.Demo/Corvus.Testing.AzureFunctions.Xunit.Demo.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <Nullable>enable</Nullable>
+
+    <!-- Disabling IDE0009, SA1101 (prefix `this`) to reduce language cruft in the test methods.
+         Disabling SA1600, SA1602 (all public types and members to be documented) because test projects need to make lots of types
+         public for technical reasaons, but they are not meant for public consumption, so XML doc comments are only
```

Comment:
I'm guessing that `reasaons` typo was copied over from our code, but since this is a new file, probably worth at least not duplicating the error!

Reply:
Resolved in b2ca266.